src/Compilers/CSharp/Test/Semantic/Semantics/StackAllocInitializerTests.cs

```diff
@@ -769,6 +769,9 @@ public void Method1()
 }", TestOptions.UnsafeReleaseDll);
 
             comp.VerifyDiagnostics(
+                // (6,31): error CS0208: Cannot take the address of, get the size of, or declare a pointer to a managed type ('int[]')
+                //         var obj1 = stackalloc int[][] { 1 };
+                Diagnostic(ErrorCode.ERR_ManagedAddr, "int").WithArguments("int[]").WithLocation(6, 31),
```

Comment:
>ERR_ManagedAddr [](start = 37, length = 15)

Can we avoid having two errors here?
Also, the message of the second error is rather confusing. But that's not related to this PR.